concent_api/core/views.py

```diff
@@ -52,6 +52,28 @@ def send(request, message):
             save_messages_to_database(data['type'], data['message_task_to_compute'], request.body)
             return HttpResponse("ACCEPTED", status = 202)
 
+    elif data['type'] == "MessageRejectReportComputedTask":
+        validate_message_reject(data['message_cannot_commpute_task'])
+
+        current_time = int(datetime.datetime.now().timestamp())
+        task_to_compute_exists  = Message.objects.filter(task_id = data['message_cannot_commpute_task']['task_id'], type = "MessageForceReportComputedTask")
+        if not task_to_compute_exists:
+            print("No MessageForceReportComputedTask in database.")
+            return HttpResponse("No MessageForceReportComputedTask in database.", status = 204)
+        rejected_message_task_to_compute = task_to_compute_exists.last()
```

Comment:
If you're using `task_to_compute_exists` for a purpose other than checking if an object exists, it's misleading to name it like that. Just call it `task_to_compute`.